Orange-Skeletons/welcome-to-nashville
# Made code modular and added styling

# Description

I updated our code to follow modular JS conventions and added styling.

Fixes # (issue)

## Type of change


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout added-styles`
4. 'hs -o' (to run server)
5. Check to see that new styles have been added and that application is still functioning

# Checklist:
